debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/xstream/LcrEventHandler.java

```diff
@@ -140,7 +142,7 @@ private void processRowLCR(RowLCR row) throws InterruptedException {
         }
         else {
             // Since the row has no chunk data, it can be dispatched immediately.
-            dispatchDataChangeEvent(row, Collections.emptyMap());
+            dispatchDataChangeEvent(row, new HashMap<>());
```

Comment:
How many elements do we expect here typically? Can we initialize this with as smaller value than the default?